specification/v4.0/OSDM-online-api-v4.0.0-draft.yml

```diff
@@ -11778,6 +11778,13 @@ components:
           items:
             $ref: '#/components/schemas/ServiceClassType'
           maxItems: 4
+        travelClassTypes:
+          description: |
+            list of travel classes where this card applies
+          type: array
+          items:
+            $ref: '#/components/schemas/TravelClassType'
```

Comment:
@CGantert345 What is TravelClassType? We only have TravelClass that also has two meaningful values + place holder value so also the maxItems could be 2 at max.